services/core-web/src/components/mine/Reports/MineReportTable.tsx

```diff
@@ -141,39 +128,39 @@ export const MineReportTable = (props) => {
       key: "mine_name",
       dataIndex: "mine_name",
       sortField: "mine_name",
-      sorter: props.isDashboardView,
-      className: hideColumn(!props.isDashboardView),
+      sorter: isDashboardView,
+      className: hideColumn(!isDashboardView),
       render: (text, record) => (
-        <div title="Mine" className={hideColumn(!props.isDashboardView)}>
+        <div title="Mine" className={hideColumn(!isDashboardView)}>
           <Link to={router.MINE_DASHBOARD.dynamicRoute(record.mine_guid)}>{text}</Link>
         </div>
       ),
     },
     {
       title:
-        props.mineReportType === Strings.MINE_REPORTS_TYPE.permitRequiredReports
+        mineReportType === Strings.MINE_REPORTS_TYPE.permitRequiredReports
           ? "Report Type"
           : "Report Name",
       key: "report_name",
       dataIndex: "report_name",
       sortField: "report_name",
-      sorter: props.isDashboardView || ((a, b) => a.report_name.localeCompare(b.report_name)),
+      sorter: isDashboardView || ((a, b) => a.report_name.localeCompare(b.report_name)),
```

Comment:
What the heck is going on here... sorter is either `true` _or_ it's a sort function? Whatever, out of scope. I'm assuming it works, nice to see it a tsx file now.

Reply:
Yeah, that's valid antD.  I've seen it before.